projects/plugins/jetpack/extensions/plugins/publicize/store/selectors.js

```diff
@@ -26,7 +31,8 @@ const DEFAULT_TWEETSTORM_MESSAGE = '\n\n' + __( 'A thread ⬇️', 'jetpack' );
  * @returns {Array} List of connections.
  */
 export function getFailedConnections( state ) {
-	return state.connections.filter( connection => false === connection.test_success );
+	const connections = getConnections( state );
```

Comment:
Does `getConnections` still take `state`?

Reply:
not anymore. updating...

Reply:
Thanks, Paul. Adressed.